confmap/provider/eecprovider/config.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Dynatrace LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package eecprovider // import "github.com/Dynatrace/dynatrace-otel-collector/confmap/provider/eecprovider/eecprovider"
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+)
+
+type config struct {
+	// A time duration that defines how frequently the provider should check the given URL for updates.
+	refreshInterval time.Duration
+
+	// A header that will be used to authenticate the provider with the EEC.
+	// We will always use Api-Token (and this can be assumed as a default if auth-file is passed),
+	// but upstream will want to support configurable headers and we should consider
+	// offering this option should upstream determine the option is required.
+	authHeader string
```

Comment:
I'd leave this out since it would also likely need configurable format for upstream. For our internal use we already know there is only 1 valid value.

Reply:
That's fine with me, done.